### ISSUE TITLE
subject is not set when using share by mail

### ISSUE THREAD

**Issue by superallumette** [MurhafSousli/ngx-sharebuttons#240]
### Bug Report or Feature Request (mark with an `x`)
```
- [x ] bug report -> please search issues before submitting
- [ ] feature request
- [ ] question
```

### OS and Version?

All
### Versions

Angular CLI: 1.7.3
Node: 9.4.0
OS: win32 x64
Angular: 6.0.0-beta.7
... animations, common, compiler, core, forms, http
... platform-browser, platform-browser-dynamic, platform-server
... router, upgrade

@angular/cdk: 6.0.0-beta-0
@angular/cli: 1.7.3
@angular/compiler-cli: 6.0.0-beta.6
@angular/flex-layout: 2.0.0-beta.12
@angular/material: 6.0.0-beta-0
@angular-devkit/build-optimizer: 0.3.2
@angular-devkit/core: 0.3.2
@angular-devkit/schematics: 0.3.2
@ngtools/json-schema: 1.2.0
@ngtools/webpack: 1.10.2
@schematics/angular: 0.3.2
@schematics/package-update: 0.3.2
typescript: 2.4.2
webpack: 3.11.0

### Repro steps

<share-buttons [theme]="'material-dark'"
    [include]="['email']"
    [title]="'Hello !'"
    >
    </share-buttons>

### Desired functionality

Subject should be set to "Hello !" in the opening mail creation window

### Mention any other details that might be useful

I've noticed that the sharer url generated is invalid.
It looks like this : mailto:?url&subject=Hello%20!&body=url
But if I change to : mailto:?subject=Hello%20!&body=url then it works.

Thanks

**Comment by MurhafSousli** [comment]
@superallumette The subject actually works! [stackblitz](https://stackblitz.com/edit/ngx-sharebuttons-xxsx5g?file=app/home/home.component.html)

**Comment by superallumette** [comment]
Hello, I'm sorry but your example does not work for me. It has to be related to Outlook then.

**Comment by MurhafSousli** [comment]
Yupp it is related to outlook, the email button implements [this SO](https://stackoverflow.com/questions/4782068/can-i-set-subject-content-of-email-using-mailto)

**Comment by superallumette** [comment]
But there is a difference with their implementation is yours.
They wrote : "mailto:?subject=look at this website&body=Hi,I found this website
and thought you might like it http://www.geocities.com/wowhtml/"
When your implementation would generate : "mailto:?http://www.geocities.com/wowhtml/&subject=look at this website&body=Hi,I found this website
and thought you might like it http://www.geocities.com/wowhtml/"

The url is pasted before the subject and the subject is prefixed with an '&'.

If I test their solution, I have a subject in outlook. With your solution, I currently don't.
